tests/syntax/function-expressions-and-better-patterns.n

```diff
@@ -90,7 +90,7 @@ let funcs = (
 	// Although redundant, you can put as many parentheses around expressions,
 	// types, and patterns as you wish. With the new function expression syntax,
 	// you can also have parentheses around the name-type pair.
-	((((arg1)): ((type1))) -> (type2) { return (arg1) }),
+	(((arg1): (type1)) -> (type2) { return (arg1) }),
```

Comment:
Uh I think `((arg1))` should be kept. 
![image](https://user-images.githubusercontent.com/22133785/132112228-9391fa35-c246-45bf-93f8-d832af9c5c92.png)

Reply:
I disagree

Reply:
Wouldn't that make the grammar more complex though